layouts/error.vue

```diff
@@ -23,7 +24,6 @@ export default {
   },
   data() {
```

Comment:
We should be using nuxt-property-decorator syntax to keep consistency

Reply:
Fixed on ad45b59e259609c193f3ed27044db1eb94e93552